bradtaniguchi/codewars-node-api
# feat/11/testing

- Add ts-jest boilerplate
- Add test step to on-push
- Add type-check test-cases
- Add API call examples


Close #11
